ckanext/bigquery/backend/bigquery.py

```diff
@@ -32,12 +33,14 @@ def _log_or_raise(self, message):
             raise Exception(message)
 
     def search(self, context, data_dict):
+        ga_api_tracker(data_dict['resource_id'])
```

Comment:
OK - seeing below we should pass i think "id" and "label" through and have the label stored in some way and maybe dataset as well so something like:

```
resource = get_resource_object(resource_id)
# todo check if package_id is uuid or name - we want it to be name
ga_api_tracker(id='resource/' + resource_id, label=resource['package_id'] + '/' + resource['name'], dataset=package_id)
```

Then change the ga_api_tracker method to take those args.

Reply:
@rufuspollock I think in our particular case (NHS project) we have precisely meaningful resource Ids (e.g. EPD_202002, PCA_2014) and the report looks informative enough and clean right now:
![Screenshot from 2020-07-30 12-19-59](https://user-images.githubusercontent.com/61929774/88906270-eb528000-d25f-11ea-88dd-1a3f289f513e.png)
Probably it's needless to add more info here?
WDYT?

Reply:
Ahhh that's because we set the resource id to be table name elsewhere i think. I'm in two minds whether we should do that but that's a refactoring so let's leave this as is for now and not make the change i'm suggesting.